src/play_time.c

```diff
@@ -0,0 +1,62 @@
+#include <nitro.h>
+
+#include "play_time.h"
+
+#define PLAYTIME_MAX_HOURS 999
+#define PLAYTIME_MAX_MINUTES 59
+#define PLAYTIME_MAX_SECONDS 59
+
+void PlayTime_Init(PlayTime *playTime)
+{
+    playTime->hours = 0;
+    playTime->minutes = 0;
+    playTime->seconds = 0;
+}
```

Comment:
Does this match as `memset(playTime, 0, sizeof(PlayTime))`, per-chance?

Reply:
It does not